### ISSUE TITLE
Certain Page Titles or Texts used as Slugs show Page not Found

### ISSUE THREAD

**Issue by kspilarski** [Automattic/wp-calypso#56236]
### Steps to reproduce the behavior

1. Go to Pages and create a new page.
2. Add a title to the page like `Avatar`.
3. On the right of the page editor, click Permalinks, make sure that slug is `avatar`.
4. Publish page.
5. Visit page link.

#### What I expected to happen

The page content will load. 

#### What actually happened

The page displayed "We couldn't find what you were looking for." message.

#### Browser / OS version

Firefox 92, macOS 11.6

#### Is this specific to the applied theme? Which one?

No.

#### Does this happen on simple or atomic sites or both?

Only simple sites. 

#### Is there any console output or error text?

"We couldn't find what you were looking for." 

Browser console doesn't show any error. 

#### Level of impact (Does it block purchases? Does it affect more than just one site?)

Affects all simple sites. 

#### Screenshot / Video: If applicable, add screenshots to help explain your problem.

Page - Avatar slug shows "We couldn't find what you were looking for." message:

<img width="1389" alt="Screen Shot 2021-09-14 at 10 45 59" src="https://user-images.githubusercontent.com/36863633/133226246-21ee6cd3-8d1d-4ca6-9477-30005ca76f2f.png">

![Screen Shot on 2021-09-14 at 10:47:58](https://user-images.githubusercontent.com/36863633/133226435-afe5080d-a507-4cfc-9fb8-4e2204a71b6f.png)

Page - using another slug like my-avatar or avatars works and displays the content:

<img width="1191" alt="Screen Shot 2021-09-14 at 10 49 23" src="https://user-images.githubusercontent.com/36863633/133226675-3da551a2-b423-44b2-9f5c-985df44de516.png">

<img width="679" alt="Screen Shot 2021-09-14 at 10 55 48" src="https://user-images.githubusercontent.com/36863633/133227628-908985d7-13c3-425a-ab0c-b75ededb9d48.png">

Post - Avatar slug works and displays the content:

<img width="1186" alt="Screen Shot 2021-09-14 at 10 51 08" src="https://user-images.githubusercontent.com/36863633/133226957-5db86494-98a9-4661-8767-5b15fac35a06.png">

<img width="699" alt="Screen Shot 2021-09-14 at 10 52 07" src="https://user-images.githubusercontent.com/36863633/133227113-4c376a4d-8eb9-4f4c-a3e2-cd561b50af18.png">

**Comment by inaikem** [comment]
Pinged @kspilarski re Bugomattic and added the issue to the HE Triage board.

**Comment by synora** [comment]
Triaged. Can replicate.
@Robertght, I'm flagging @Automattic/flow-patrol-create. This issue affects all simple sites.

**Comment by arcangelini** [comment]
This is likely due to the way that WordPress.com used to serve avatars. It was a request made to the URL `$host/avatar`. When we switched to Gravatar I think we kept the `/avatar` to prevent it from breaking everywhere.

It looks like this was spotted a couple of years ago but fell through the cracks: p1575462107235800/1575405208.234200-slack-C02FMH4G8

I updated the Fieldguide post and made the change on WPCOM. Here is the diff: D67565-code

Should be reviewed and published by end of the day.
